routers/api/v1/api.go

```diff
@@ -514,6 +514,7 @@ func RegisterRoutes(m *macaron.Macaron) {
 		m.Post("/markdown/raw", misc.MarkdownRaw)
 		m.Group("/settings", func() {
 			m.Get("/allowed_reactions", misc.SettingGetsAllowedReactions)
```

Comment:
If we are going to do general settings, I think before anybody uses it you should change allowed_reactions to a more generic ```ui``` and do the same thing (choose ui just to match config ini) and then people can query settings/ui for ui settings

Reply:
ok if we go this way I'll create a new section called settings

Reply:
looks like there is already /settings -- under settings could just have ones that match config ini (repository, server, ui, etc...) and it would map to appropriate values in config ini